abi/testdata/abi.abi.json

```diff
@@ -0,0 +1,62 @@
+{
+    "actions": [
+        {
+            "id": 255,
+            "name": "VM",
+            "types": [
+                {
+                    "name": "VM",
+                    "fields": [
+                        {
+                            "name": "actions",
+                            "type": "[]Action"
+                        }
+                    ]
+                },
```

Comment:
We synced on this offline - but we discussed seeing if we could define types at the ABI level, instead of the action-level so that you don't have to re-define types that are shared between actions.

Reply:
Done!